stories/base/FormFields.stories.js

```diff
@@ -312,5 +313,103 @@ Timestamp.argTypes = {
     readOnly,
 };
 
+export const SearchField = ({ label, helperText, required }) => {
+    const CustomOption = (option) => (
+        <ListItemText primary={option.label} secondary={option.iri} />
+    );
+    const searchOptions = [
+        {
+            id: "envo:class:http://purl.obolibrary.org/obo/ENVO_01000955",
+            iri: "http://purl.obolibrary.org/obo/ENVO_01000955",
+            label: "desert tree line ecotone",
+            ontology_prefix: "ENVO",
+        },
+        {
+            id: "envo:class:http://purl.obolibrary.org/obo/ENVO_01000956",
+            iri: "http://purl.obolibrary.org/obo/ENVO_01000956",
+            label: "desert-alpine tree line ecotone",
+            ontology_prefix: "ENVO",
+        },
+        {
+            id: "envo:class:http://purl.obolibrary.org/obo/ENVO_01000179",
+            iri: "http://purl.obolibrary.org/obo/ENVO_01000179",
+            label: "desert biome",
+            ontology_prefix: "ENVO",
+        },
+        {
+            id: "envo:class:http://purl.obolibrary.org/obo/ENVO_01001780",
+            iri: "http://purl.obolibrary.org/obo/ENVO_01001780",
+            label: "desert ecosystem",
+            ontology_prefix: "ENVO",
+        },
+        {
+            id: "envo:class:http://purl.obolibrary.org/obo/ENVO_03500006",
+            iri: "http://purl.obolibrary.org/obo/ENVO_03500006",
+            label: "food desert",
+            ontology_prefix: "ENVO",
+        },
+        {
+            id: "envo:class:http://purl.obolibrary.org/obo/ENVO_01000185",
+            iri: "http://purl.obolibrary.org/obo/ENVO_01000185",
+            label: "montane desert biome",
+            ontology_prefix: "ENVO",
+        },
+        {
+            id: "envo:class:http://purl.obolibrary.org/obo/ENVO_01000186",
+            iri: "http://purl.obolibrary.org/obo/ENVO_01000186",
+            label: "polar desert biome",
+            ontology_prefix: "ENVO",
+        },
+        {
+            id: "envo:class:http://purl.obolibrary.org/obo/ENVO_01000182",
+            iri: "http://purl.obolibrary.org/obo/ENVO_01000182",
+            label: "temperate desert biome",
+            ontology_prefix: "ENVO",
+        },
+        {
+            id: "envo:class:http://purl.obolibrary.org/obo/ENVO_01000183",
+            iri: "http://purl.obolibrary.org/obo/ENVO_01000183",
+            label: "tropical desert biome",
+            ontology_prefix: "ENVO",
+        },
+        {
+            id: "envo:class:http://purl.obolibrary.org/obo/ENVO_01000184",
+            iri: "http://purl.obolibrary.org/obo/ENVO_01000184",
+            label: "subtropical desert biome",
+            ontology_prefix: "ENVO",
+        },
+    ];
+    return (
+        <TestForm initialValue="subtropical desert biome">
```

Comment:
Sorry, this `TestForm` is kind of weird, but now that the `required` boolean control is used in the Field, this `TestForm` should also use the `required` prop:
```suggestion
        <TestForm required={required} initialValue="subtropical desert biome">
```